collect_app/src/main/res/values/strings.xml

```diff
@@ -442,8 +442,8 @@
 <string name="invalid_rsa_public_key">Invalid RSA public key.</string>
 <string name="encryption_error_message">%s Form has not been saved as finalized.</string>
 <string name="analytics_preferences">Analytics Options</string>
-<string name="enable_analytics">Enable Analytics</string>
-<string name="enable_analytics_summary">Uncheck to disable Analytics</string>
+<string name="analytics">Enable Analytics</string>
```

Comment:
Now the preferences screen doesn't use the word analytics at all so I think this should say "usage data".